### ISSUE TITLE
Radio Button forms

### ISSUE THREAD

**Issue by ajoshdee** [xinthink/react-native-material-kit#109]
HI, I'm creating a form which utilizes the radio button in groups of 5. Can you help me implement this. I have multiple questions in my form.

**Comment by xinthink** [comment]
What's problem are you facing? Would you please post the code snippet here?

**Comment by ajoshdee** [comment]
For example if I have 100 questions. Do I have to create 100 instances of `this.radioGroup` = new MKRadioButton.Group();`?

**Comment by xinthink** [comment]
There're serval answers for a single question? 
So I think there should be a group for each question.

**Comment by ajoshdee** [comment]
And is it possible to just set the number of radio buttons, instead of calling it every time I need a radio button?

**Comment by ajoshdee** [comment]
So are you suggesting that I have a hundred variables for 100 groups?

**Comment by xinthink** [comment]
Yes. I think the way you use the Radio buttons should just like what you will do using html tags.
You can write a component yourself to wrap multiple Radio buttons.

**Comment by xinthink** [comment]
Sorry, I didn't mean you should have hundreds of variables. But there's should be hundreds of groups.
You can make it local variables, like using a method to do the job, or write a custom component.

**Comment by ajoshdee** [comment]
Ah yes but , each `new MKRadioButton.Group()` needs to be assigned to a prop. How do I go about this? It's easy if I could dynamically change the group by name but group prop accepts an object property.

**Comment by xinthink** [comment]
Sorry for replying it so late. But I wrote an example below, hope it may help.

```jsx
// say you have a ListView for the questions
<ListView
  dataSource={this.state.questionList}
  renderRow={this.renderQuestion}
/>

// rendering a single question
// we create a group for each question, for it must be single selection, if you're using RadioButtons
renderQuestion(question) {
  const group = new MKRadioButton.Group();
  const answers = [];
  question.answers.forEach((answer) => answers.push(<MKRadioButton group={group}/>));
  return <View>{answers}</View>;
}
```
